Jenkinsfile

```diff
@@ -110,6 +110,11 @@ pipeline{
 								sh "scripts/dev_cli.sh tests --integration --libc musl"
 							}
 						}
+						stage ('Run metrics tests') {
+							steps {
+								sh 'scripts/dev_cli.sh tests --metrics'
```

Comment:
How long does this take? The aarch64 builder is already a "long pole" in our CI. I don't think we should do this unless we can get a 2nd system for that purpose.

Reply:
about ~20min on my machine.

Reply:
Done